src/index.tsx

```diff
@@ -87,26 +120,25 @@ function PrefabProvider({
     if (mostRecentlyLoadingContextKey.current === undefined) {
       mostRecentlyLoadingContextKey.current = contextKey;
 
-      const initOptions: Parameters<typeof prefab.init>[0] = {
+      if (!apiKey) {
+        throw new Error("PrefabProvider: apiKey is required");
+      }
+
+      const initOptions: Parameters<typeof prefabClient.init>[0] = {
         context,
+        ...settings,
         apiKey,
```

Comment:
`apiKey` is in the settings object now, so seems like we want to delete this line

Reply:
Sounds good!

Reply:
Ah typescript gets made without it (beacause then it an be undefined). I'm leaving it in but adding a comment.